pkg/services/chgm/chgm.go

```diff
@@ -161,65 +254,30 @@ func (i InvestigateInstancesOutput) String() string {
 		msg += fmt.Sprintf("\nInstance IDs: '%v' \n", ids)
 	}
 
-	summary, ok := i.LimitedSupportReason.GetSummary()
-	if !ok {
-		summary = "No summary was posted"
-	}
-	details, ok := i.LimitedSupportReason.GetDetails()
-	if !ok {
-		details = "No details were posted"
-	}
-	msg += fmt.Sprintln("\nLimited Support reason sent:")
-	msg += fmt.Sprintf("- Summary: '%s'\n", summary)
-	msg += fmt.Sprintf("- Details: '%s'\n", details)
+	// summary, ok := i.LimitedSupportReason.GetSummary()
+	// if !ok {
+	// 	summary = "No summary was posted"
+	// }
+	// details, ok := i.LimitedSupportReason.GetDetails()
+	// if !ok {
+	// 	details = "No details were posted"
+	// }
+	// msg += fmt.Sprintln("\nLimited Support reason sent:")
+	// msg += fmt.Sprintf("- Summary: '%s'\n", summary)
+	// msg += fmt.Sprintf("- Details: '%s'\n", details)
```

Comment:
**chore:** Remove or uncomment dead code.

Reply:
i uncommented it for now, i think we might want to improve CADs info ( incident notes/title change) for Primary in general
but we can handle that in a follow up if we need to

Reply:
update: I changed it back to post limited support reasons on the pagerduty notes